docs/antrea-network-policy.md

```diff
@@ -409,6 +410,81 @@ spec:
         enableLogging: true
 ```
 
+#### ACNP for multicast
+
+ACNP for IGMP query
+
+```yaml
+apiVersion: crd.antrea.io/v1alpha1
+kind: ClusterNetworkPolicy
+metadata:
+  name: acnp-with-igmp-query-drop
+spec:
+    priority: 5
+    tier: securityops
+    appliedTo:
+      - podSelector:
+          matchLabels:
+            app: mcjoin6
+    ingress:
+      - action: Drop
+        from:
+        protocols:
+        - igmp:
+            igmpType: 0x11
+            groupAddress: 224.0.0.1
+        name: dropIGMPQuery
+        enableLogging: false
```

Comment:
I think we don't support logging for IGMP and multicast? then don't put the field in the examples which may mislead users that they can enable it.

Reply:
done